tests/storage/merge_test.py

```diff
@@ -236,17 +236,23 @@ def test_block_cow(self, monkeypatch, base, top, expected):
             def fake_measure(image, format=None, output_format=None,
                              backing=True, is_block=False, base=None,
                              unsafe=False):
+                bitmaps = 0
                 # Make sure we are called with the right arguments.
-                assert image == top_vol.getVolumePath()
+                if image == top_vol.getVolumePath():
+                    assert base == base_vol.getVolumePath()
+                    # Top measure returns bitmaps.
+                    bitmaps = 1 * MiB
+                else:
+                    assert image == base_vol.getVolumePath()
+                    assert base is None
                 assert format == qemuimg.FORMAT.QCOW2
                 assert output_format == qemuimg.FORMAT.QCOW2
                 assert backing
                 assert is_block
-                assert base == base_vol.getVolumePath()
                 assert not unsafe
 
                 # Return fake response.
-                return {"required": 2048 * MiB, "bitmaps": 1 * MiB}
+                return {"required": 2048 * MiB, "bitmaps": bitmaps}
```

Comment:
This could be:

    return {"required": 2048 * MiB, "bitmaps": fake_bitmap_size[image]}